src/server/ApiRoutes/Search.js

```diff
@@ -28,11 +29,7 @@ const createAPIQuery = basicQuery({
 });
 
 const nyplApiClientCall = (query, urlEnabledFeatures = []) => {
-  const requestOptions =
-    appConfig.features.includes('on-site-edd') ||
-    urlEnabledFeatures.includes('on-site-edd')
-      ? { headers: { 'X-Features': 'on-site-edd' } }
-      : {};
+  const requestOptions = noOnsiteEddCheck(appConfig)
```

Comment:
Perhaps we can have a combined argument of the two lists sent as an argument.
Thoughts?

Reply:
what two lists?

Reply:
`appConfig.features` && `urlEnabledFeatures`